src/app/3d/qgspoint3dsymbolwidget.cpp

```diff
@@ -194,10 +217,15 @@ QgsPoint3DSymbol QgsPoint3DSymbolWidget::symbol() const
       vm[QStringLiteral( "radius" )] = spinRadius->value();
       vm[QStringLiteral( "minorRadius" )] = spinMinorRadius->value();
       break;
-    case 6:  // model
+    case 6:  // 3d model
       vm[QStringLiteral( "model" )] = lineEditModel->text();
       vm[QStringLiteral( "overwriteMaterial" )] = cbOverwriteMaterial->isChecked();
       break;
+    case 7:  // billboard
+      QgsDebugMsg( QStringLiteral( "Set billboard from symbol." ) );
+      QgsMarkerSymbol *billboardSymbol = static_cast<QgsMarkerSymbol *>( btnChangeSymbol->symbol() ) ;
+      sym.setBillboardSymbol( billboardSymbol );
```

Comment:
Possible ownership issue here, depending on whether or not setBillboardSymbol takes ownership or internally makes a clone. 

(Ideally it should take ownership and follow the patterns from other parts of QGIS). In which case, this should use QgsSymbolButton::clonedSymbol instead.

Reply:
Done, use clonedSymbol now